cmake/GridToolsConfig.cmake.in

```diff
@@ -24,7 +24,11 @@ set(GRIDTOOLS_HAS_BACKEND_MC @GT_ENABLE_BACKEND_MC@)
 set(GRIDTOOLS_HAS_BACKEND_X86 @GT_ENABLE_BACKEND_X86@)
 set(GRIDTOOLS_HAS_BACKEND_NAIVE @GT_ENABLE_BACKEND_NAIVE@)
 
-if (GRIDTOOLS_ENABLE_PERFORMANCE_METERS)
+if(@CUDA_AVAILABLE@)
+    set(GRIDTOOLS_CUDA_COMPILATION_TYPE "@GT_CUDA_COMPILATION_TYPE@")
+endif()
+
+if(GRIDTOOLS_ENABLE_PERFORMANCE_METERS)
```

Comment:
Unrelated to this PR, but I think we shouldn’t put enable performance meters in the config. But probably it is not worth fixing as the meters will disappear in 2.0 anyway...

Reply:
yes I agree that might be something that we just want to document because we might want to enable it for certain targets... let's wait for 2.0 and reiterate the config then, ok?